artemis/readme.adoc

```diff
@@ -0,0 +1,36 @@
+== Camel Example Spring Boot and ActiveMQ AMQP
+
+This example shows how to work with a simple Apache Camel application using Spring Boot and Apache ActiveMQ Artemis.
+
+=== Preparing ActiveMQ artemis brokers
+
+From Apache ActiveMQ you can download the broker as a `.zip` or `.tar.gz` file.
+
+Unzip/tar the archive, and start a terminal.
+
+Change directory to the unzipped directory and start the broker.
+
+    bin/artemis run
```

Comment:
Is it still needed if you use the embedded mode?

Reply:
done Removed